### PR TITLE
Check for plugins.dependencies

### PR DESCRIPTION
**Pull request by squeakymouse**
Allow `plugins.dependencies` to override `get_external_interfaces`, `get_external_interfaces_read_only`, `verify_authentication`

### CODE REVIEW

**Review by phil-scale — COMMENTED** [review]
Is there supposed to be a change to `setup.py` that's not checked in?

**Comment by song-william** [comment]
> Is there supposed to be a change to setup.py that's not checked in?

What type of change are you thinking of?

File: server/Dockerfile [hunk 1]
```diff
@@ -44,6 +44,7 @@ RUN pip install -r requirements.txt --no-cache-dir
 RUN pip install -r requirements_override.txt --no-cache-dir
 COPY server/pyproject.toml /workspace/server/pyproject.toml
 COPY server/setup.py /workspace/server/setup.py
+COPY server/service_configs /workspace/server/service_configs
```
Comment (song-william) [inline server/Dockerfile:47]: What did this change enable?

**Comment by phil-scale** [comment]
Superceded by https://github.com/scaleapi/llm-engine/pull/214

**Pull request closed without merging**